lib/matplotlib/dates.py

```diff
@@ -1926,6 +1926,8 @@ class _rcParam_helper:
     @classmethod
     def set_converter(cls, s):
         """Called by validator for rcParams date.converter"""
+        if s not in ['concise', 'auto']:
```

Comment:
`s` is just a string, not a dict.

Reply:
`_check_in_list` doesn't check `dict`s?

Reply:
It would work:
```python
cbook._check_in_list(['concise', 'auto'], s=s)
```
The example in the docstring is misleading; as far as I can see, this would rarely if ever be called with more than one keyword argument.  The reason it requires a keyword argument is that this supplies a name for the variable when reporting the error.